samples/noxfile.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2019 Google LLC
```

Comment:
Should this be 2020?

Reply:
🤔 This is a lightly edited version of `noxfile-template.py` from python-docs-samples. I'm not sure what the policy is around copyright years in this case

Reply:
If you copy, keep the original date. If you want to add the new year, you'd better have multiple years.